docs/spec_draft.md

```diff
@@ -281,16 +285,16 @@ the IEEE-754 specification. For boolean element type, the behavior is same as
 
 ### Inputs
 
-| Name  | Type                         |
-|-------|------------------------------|
-| `lhs` | tensor of any supported type |
-| `rhs` | tensor of any supported type |
+| Name  | Type                                                        |
+|-------|-------------------------------------------------------------|
+| `lhs` | tensor of integer, boolean, floating-point, or complex type |
+| `rhs` | tensor of integer, boolean, floating-point, or complex type |
```

Comment:
Given that above we're saying that token types cannot be element types, we don't need this change (and a couple dozen of similar changes below). This is similar to how we know that "any supported type" in "tensor of any supported type" cannot refer to function types.